Created a desktop version branch to implement media query for size 768px and above

AmitKlohan opened the pull request:
I've a completed the desktop version of my portfolio project and fulfilled the requirements of the project.

- no linters error 
- used correct Github Flow
- I've used best practices in my HTML and CSS
- Add a media query to run for screen size 768px and wider
- Incorporate changes to make this look like the Figma design template as much as possible
- Make uses of  Flexbox and Grid elements

AmitKlohan committed:
add a media query for screen size 768px and above

AmitKlohan committed:
set menu icon display none for desktop

AmitKlohan committed:
add style to navbar in header

AmitKlohan committed:
change card flex direction to row and set the cards order as per desktop

AmitKlohan committed:
set about myself section flex direction row with order: 6

AmitKlohan committed:
set about myself child elements flex direction to column

AmitKlohan committed:
add style to form

AmitKlohan committed:
add display none to bottom line and add  padding-bottom: 2rem to class card-sw

AmitKlohan committed:
set background image for desktop first page

AmitKlohan committed:
fix linters bug

zhadier commented on style.css:
```diff
@@ -344,3 +344,167 @@ header {
   align-self: center;
 }
+
+@media only screen and (min-width: 768px) {
+  .head-section {
+    flex-direction: column;
+    background-image: url("desktop_header_bg.png");
```
- [ ] Kindly add a fixed `max-width` to adjust the size for your logo in the toolbar, right now on some of the larger screen sizes, the logo just gets too big and takes up too much of the screen which does not look as good as it is on some of the smaller screen sizes.

AmitKlohan replied:
done

zhadier commented on style.css:
```diff
@@ -344,3 +344,167 @@ header {
+    margin-right: 0;
+  }
+
+  .card4 {
+    flex-direction: row-reverse;
+    order: 3;
+  }
```
- [ ] One of your cards seems to have a disabled button as you can see below. Kindly enable it so it conforms to the Figma design parameters.

![image](https://user-images.githubusercontent.com/90556221/178960129-fa38aba2-ef62-44d7-b8ab-03959cb420ac.png)

AmitKlohan replied:
Done

zhadier commented on style.css:
```diff
@@ -344,3 +344,167 @@ header {
+    width: 45%;
+  }
+
+  .about-myself-second-section {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    width: 55%;
+  }
```
- [ ] Kindly add a little padding to the skills images inside your about section so they look similar to the Figma design ( there's a gap between the image and the border )

> Your Design
![image](https://user-images.githubusercontent.com/90556221/178960640-19e6ee21-03fc-4b87-9e70-6d9840e13902.png)

> Figma Design
![image](https://user-images.githubusercontent.com/90556221/178960721-df40e948-f63d-48b1-b139-f08426b1249d.png)

- [ ] Additionally, please also ensure that the headings for your skills have the same color as in the Figma template

AmitKlohan replied:
done

zhadier commented on style.css:
```diff
@@ -344,3 +344,167 @@ header {
+    margin-top: 0;
+  }
+
+  .contact-me {
+    order: 7;
+  }
```
- [ ] There seems to be a small gap between your about and contact sections which is not present in the Figma template, kindly remove this gap.

![image](https://user-images.githubusercontent.com/90556221/178961366-7ac13f80-1db4-482e-8741-47ed0ca522ff.png)

AmitKlohan replied:
done

zhadier commented on style.css:
```diff
@@ -344,3 +344,167 @@ header {
+    order: 7;
+  }
+
+  form {
+    width: 45rem;
+    margin: 0 auto;
+  }
```
- [ ] What do you think about styling the `::placeholder` selector in the `textarea` field and introducing a little padding so it looks similar to the Figma design? Right now the placeholder is attached to the border and it doesn't look quite right visually. Please do implement this feature.

![image](https://user-images.githubusercontent.com/90556221/178961955-136f7172-46ec-4086-8e5a-a5ba1fd338c6.png)

AmitKlohan replied:
done

zhadier reviewed (changes requested):
# Hi @AmitKlohan  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 

<img src="https://media4.giphy.com/media/5wWf7GR2nhgamhRnEuA/giphy.gif"/>

## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description
✔️ You have done well matching your design to the original Figma design
✔️ You have used CSS grid and flexbox well for aligning the elements

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AmitKlohan committed:
fix the logo size: 8rem and add style to card, contact me page

AmitKlohan commented:
Implementation done as suggested above by CR Mr @zhadier 
- Set the logo image size max-width: 8rem.
- Enabled the button.
- Add padding-left: 1rem to skills section.
- Changed the colour as per Figma design.
- Removed the gap between about me and contact us page.

omarsalem7 reviewed (approved):
Hi @AmitKlohan 😊, 
# STATUS: APPROVED! 🥇 💯 

![giphy](https://media.giphy.com/media/n5VaQoW39Z9S0/giphy.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
